packages/deno/mod.ts

```diff
@@ -0,0 +1,160 @@
+// Derived from https://github.com/denoland/deno_std/blob/main/log/handlers.ts
+import * as log from 'https://deno.land/std@0.203.0/log/mod.ts';
+
+import type { LogRecord } from 'https://deno.land/std@0.203.0/log/logger.ts';
+import {
+  LevelName,
+  LogLevels,
+} from 'https://deno.land/std@0.203.0/log/levels.ts';
+
+import { SeverityNumber, logs } from 'npm:@opentelemetry/api-logs@0.43.0';
+import { OTLPLogExporter } from 'npm:@opentelemetry/exporter-logs-otlp-http@0.43.0';
+import { OTLPExporterNodeConfigBase } from 'npm:@opentelemetry/otlp-exporter-base@0.43.0';
+import {
+  Resource,
+  detectResourcesSync,
+  envDetectorSync,
+  hostDetectorSync,
+  osDetectorSync,
+  processDetector,
+} from 'npm:@opentelemetry/resources@1.17.0';
+import {
+  BatchLogRecordProcessor,
+  ConsoleLogRecordExporter,
+  BufferConfig,
+  LoggerProvider,
+} from 'npm:@opentelemetry/sdk-logs@0.43.0';
+
+import type { Logger } from 'npm:@opentelemetry/api-logs@0.43.0';
+import type { Attributes } from 'npm:@opentelemetry/api@1.6.0';
+
+// https://github.com/open-telemetry/opentelemetry-specification/blob/fc8289b8879f3a37e1eba5b4e445c94e74b20359/specification/logs/data-model.md#displaying-severity
+const OTEL_SEVERITY_NAME_MAP = {
+  0: '', // XXX: This is not to spec
```

Comment:
@wrn14897 I forgot to ask you this - but what's our current behavior for a undefined level? Should we keep it as empty string or something else? It's undefined in the otel spec and I'm not sure if we do something about this elsewhere.

We do need _something_ here though I think to keep the lookup table complete.

Reply:
interesting, I don't recall anything regarding undefined level but I'm going to make sure we address that

Reply:
I think the default is 'info' if severity text is undefined or empty